portfolio/src/main/java/com/google/sps/data/TimeRangeGroup.java

```diff
@@ -27,4 +28,11 @@
    * @return the newly modified collection of time ranges.
    */
   public Collection<TimeRange> deleteTimeRange(TimeRange timeRangeToDelete);
+
+  /**
+   * Sorts the collection of time ranges based on given comparator and a boolean value that denotes
+   * if the sorting is in ascending order. The comparator and the boolean value are passed to the
+   * constructor of the class.
+   */
+  public void sortTimeRanges(Comparator<TimeRange> comparator, boolean ascending);
```

Comment:
I don't think you need both a comparator and the `ascending` boolean. If the caller already has a `Comparator` they can reverse it themselves before passing it in.

Reply:
Updated.